playbooks/roles/mattermail/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+- name: create mattermail group
+  group:
+    name: mattermail
+
+- name: create mattermail user
+  user:
+    name: mattermail
+    group: mattermail
+    home: "{{ MATTERMAIL_HOME }}"
+    system: yes
+
+- name: Install git and golang packages
+  apt:
+    name:
+      - git
+      - golang
+    update_cache: yes
+
+- name: build Mattermail binary
+  command: go get {{ MATTERMAIL_REPO }}
+  args:
+    creates: "{{ MATTERMAIL_HOME }}/bin/mattermail"
+  become_user: mattermail
+  environment:
+    GOPATH: "{{ MATTERMAIL_HOME }}"
+
+- name: create config
+  template:
+    src: config.json
```

Comment:
nit: use the .j2 extension for templates

Reply:
👍